src/components/Card/Card.css

```
@@ -1,8 +1,15 @@
 .Card {
-  color: var(--color-primary);
-  background-color: #eee;
+  color: darkred;
+  font-weight: 900;
+  background: none;
   padding: 20px;
   margin: 20px;
   text-align: center;
   border-radius: 15px;
 }
+
+.moreInformation {
```

How about BEM? 
```suggestion
.Card__button {
```
Or something like that?